Resources/config/doctrine/TermTaxonomy.orm.xml

```diff
@@ -25,5 +25,9 @@
         <many-to-one field="term" target-entity="Ekino\WordpressBundle\Entity\Term" inversed-by="taxonomies">
             <join-column name="term_id" referenced-column-name="term_id" nullable="false" />
         </many-to-one>
+
+        <many-to-one target-entity="Ekino\WordpressBundle\Entity\Term" field="parent">
```

Comment:
Just for readability, can you please move the `field="parent"` as the first attribute please? thank you ;)